sales/send_quotations/get_signature_to_validate.rst

```diff
@@ -2,31 +2,51 @@
 Get a signature to confirm an order
 ===================================
 
-You can use online signature to get orders automatically confirmed. Both
-you and your customer will save time by using this feature compared to a
-traditional process.
+Online signatures are like electronic "fingerprints". With their use on Odoo, you will get
+automatically confirmed orders. Indeed, you and your customers will save a lot of time by using this
+feature compared to a traditional process.
 
-Activate online signature
-=========================
+How to enable online signature
```

Comment:
Content: I'd keep the  infinitive form here as that's what we generally do for headings. It avoids to start every heading with "How to ...".

Reply:
Also same review, don't like "How to" for docs